Apps/Src/ReadTritium.c

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2021 UT Longhorn Racing Solar */
+
+#include "ReadTritium.h"
+
+void Task_ReadTritium(void* p_arg){
+
+	OS_ERR err;
+	CPU_TS ts;
+
+	OSSEMPend(&READTritium_Sem4, 0, OS_OPT_PEND_BLOCKING, &ts, &err);
+
+	CANbus_Init();
+
+	uint8_t *message;
+	ErrorStatus status = CANbus_Read(message);
+
+	if(message){

```

Comment:
We shouldn't be conditioning this on `message` since that's just a pointer. I think it should be conditioned on the `status`